homeassistant/generated/integrations.json

```diff
@@ -6308,6 +6308,12 @@
       "config_flow": true,
       "iot_class": "cloud_polling"
     },
+    "weatherkit": {
+      "name": "Apple WeatherKit",
+      "integration_type": "hub",
+      "config_flow": true,
+      "iot_class": "cloud_polling"
```

Comment:
Should be added to the apple brand as well

Reply:
Where would I add that? Since this is a generated file I assume I'd need to add it elsewhere, but I am looking in the manifests of other integrations in the Apple brand (`homekit`, `homekit_controller`) and I'm not seeing anything there

Reply:
Nevermind, found the relevant documentation: https://developers.home-assistant.io/docs/creating_integration_brand